packages/collabswarm-yjs/bin/collabswarm-yjs-d.ts

```diff
@@ -14,14 +14,14 @@ const keychain = new YjsKeychainProvider();
 crypto.subtle
   .generateKey(
     {
-      name: "AES-GCM",
-      length: 256,
+      name: "ECDSA",
+      namedCurve: "P-384",
     },
     true,
-    ["encrypt", "decrypt"]
+    ["sign", "verify"]
```

Comment:
Maybe we can discuss more offline - curious about background, also why bin is in repo